README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is the framework of your website and should be organized and used in a fashion that makes it easy to follow.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+The "display:block;" tag takes up a whole line where as the "display:inline;" makes it so you can make things side by side on your website.
```

Comment:
great job defining the two, many people struggle with this concept but you got it down